src/lib_ccx/hardsubx.c

```diff
@@ -102,11 +102,16 @@ int hardsubx_process_data(struct lib_hardsubx_ctx *ctx)
 	
 	mprint("Beginning burned-in subtitle detection...\n");
 
-	if(ctx->tickertext)
+	if(ctx->tickertext && ctx->russian)
+		hardsubx_process_frames_tickertext_russian(ctx, enc_ctx);
+	else if(ctx->tickertext)
 		hardsubx_process_frames_tickertext(ctx, enc_ctx);
+	else if(ctx->russian)
```

Comment:
ctx->russian without tickertext being a separate function makes no sense. We can simply set the ocrlang parameter to rus.

Reply:
It was to run just simple basic frameskip if hardsubx is activated, but noted :)